sdk/python/feast/infra/offline_stores/offline_utils.py

```diff
@@ -130,6 +133,13 @@ def get_feature_view_query_context(
         event_timestamp_column = feature_view.input.event_timestamp_column
         created_timestamp_column = feature_view.input.created_timestamp_column
 
+        min_event_timestamp = None
+        if feature_view.ttl:
+            min_event_timestamp = (
+                entity_df_timestamp_range[0] - feature_view.ttl
+            ).isoformat()
```

Comment:
Another reason for test failing is that here we have timezone-aware timestamps, which are not very well handled by Redshift when passed into query. That probably has some explanation, like we not using timezone-aware types in Redshift tables, but I think keeping all timestamps in UTC is a much better solution.
So let's convert here to UTC.

Reply:
Correction, it's not all the time timezone-aware, but it is in some cases.

Reply:
Changed this!